common.mk

```diff
@@ -246,7 +246,6 @@ $(SFC_MFC_TARGETS) &: $(TAPEOUT_CLASSPATH_TARGETS) $(FIRRTL_FILE) $(FINAL_ANNO_F
 	@if [ $(shell cat $(SFC_LEVEL)) = low ]; then cat $(TMP_DIR)/unnec-anno-deleted2.sfc.anno.json > $(SFC_ANNO_FILE) && rm $(TMP_DIR)/unnec-anno-deleted.sfc.anno.json && rm $(TMP_DIR)/unnec-anno-deleted2.sfc.anno.json; fi
 	firtool \
 		--format=fir \
-		--dedup \
```

Comment:
What happens when you remove this? We still want dedup on modules we intend to dedup.

Reply:
Dedup will always happen now. This flag is removed in 1.57.0

Reply:
https://github.com/llvm/circt/blob/c823db7a53131ced67668839a764bd7af84cc1b8/lib/Firtool/Firtool.cpp#L90

Reply:
Nice